src/components/qas/styles.module.scss

```diff
@@ -13,3 +13,9 @@
     box-shadow: none;
   }
 }
+
+.label-with-number {
+  p:first-child() {
```

Comment:
don;'t think we need the `:global` in these cases because there is no global classname on our p tags to override...and I mostly tested this hypothesis...